src/types.ts

```diff
@@ -421,6 +421,13 @@ export interface GetSwapQuoteResponse extends SwapQuoteResponsePartialTransactio
     estimatedGasTokenRefund: BigNumber;
     allowanceTarget?: string;
     quoteReport?: QuoteReport;
+    prices?: SourceComparison[];
+}
+
+export interface SourceComparison {
+    name: ERC20BridgeSource | '0x';
```

Comment:
is this just to support saying `0x` instead of `Native`?

could we leave this as `name: ERC20BridgeSource;` and do the rename of `Native` -> `0x` after calling the function that returns the `SourceComparison`?  That would feel more natural to me

Reply:
thanks for this change!